yarn-project/p2p/src/services/libp2p/libp2p_logger.ts

```diff
@@ -0,0 +1,78 @@
+import { createLogger } from '@aztec/foundation/log';
```

Comment:
This logger shims our pino logger into libp2p's logging stack. All logs get tagged at trace by default as it is extremely noisy